CreateMassiveMaximumLength8.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class CreateMassiveMaximumLength8 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input the length of the array which has to be in <3-8> range ");
+        int n = sc.nextInt();
+        if (n < 3 || n > 8) {
+            System.out.println("Not valid length for this array");
+            System.exit(1);
+        }
+        int Massive[] = new int[n];
+        System.out.println("Input two whole numbers ");
+        int a = sc.nextInt();
+        int b = sc.nextInt();
+        Massive[(n / 2 - 1)] = a;
```

Comment:
I mentioned in the exercise description that you need to figure out min and max values of `a` and `b` and depending on that do the assignments. Here you haven't taken it into account. Fix that.

Reply:
My bad, 
didn't read carefully.
Fixed.